Documentation/ApiOverview/Routing/AdvancedRoutingConfiguration.rst

```diff
@@ -69,7 +69,12 @@ and resolving the route.
 Enhancers
 =========
 
-There are two types of enhancers: decorators and route enhancers. A route
+..  tip::
+
+    See :ref:`routing-terminology` for an introduction to the used
+    terminology here.
+
+There are two types of enhancers: route decorators and route enhancers. A route
```

Comment:
I would like to avoid "PHP internal" technology here but instead try to describe this without talking about Interfaces and extended classes or so. It's complicated enough as it is, so maybe not get too dirty with Code, if possible